main.js

```diff
@@ -94,39 +79,44 @@ client.on('ready', async () => {
                         }
                     }
                     // Write the new collectors.json
-                    json = JSON.stringify(obj);
-                    fs.writeFile('./collectors.json', json, 'utf8', function(err) {
+                    const json = JSON.stringify(obj);
+                    fs.writeFile('./collectors.json', json, 'utf8', function (err) {
                         if (err) throw err;
-                        console.log("collectors.json Updated")
+                        console.log('collectors.json Updated');
                     });
                 }
-            }
-            , 1000);
+            }, 1000);
         }
     });
-})
+});
 
-fs.readdir("./events/", (err, files) => {
-  if (err) return console.error(err);
-  files.forEach(file => {
-      const event = require(`./events/${file}`);
-      let eventName = file.split(".")[0];
-      console.log(`Loading event ${eventName}`);
-      client.on(eventName, event.bind(null, client));
+fs.readdir('./events/', (err, files) => {
+    if (err) return console.error(err);
+    files.forEach((file) => {
+        const event = require(`./events/${file}`);
+        let eventName = file.split('.')[0];
+        console.log(`Loading event ${eventName}`);
+        client.on(eventName, event.bind(null, client));
     });
 });
 
 client.commands = new Collection();
 
-fs.readdir("./commands/", (err, files) => {
+fs.readdir('./commands/', (err, files) => {
     if (err) return console.error(err);
-    files.forEach(file => {
-        if (!file.endsWith(".js")) return;
+    files.forEach((file) => {
+        if (!file.endsWith('.js')) return;
         let props = require(`./commands/${file}`);
-        let commandName = file.split(".")[0];
+        let commandName = file.split('.')[0];
         console.log(`Loaded ${commandName}`);
         client.commands.set(commandName, props);
     });
 });
 
 client.login(token.token);
+
+const checkMessage = async (id, channelid) => {
+        const channel = await client.channels.cache.fetch(channelid);
+        const channelMessages = await channel.messages.fetch()
```

Comment:
We are also missing input into the fetch function because we are searching for 1 specific id

Reply:
fixed